convince-your-company-to-support-coup.html

```diff
@@ -8,49 +8,73 @@ <h1>
 </h1>
 
 <p>
-  For co.up to survive, it’s going to take recurring donations, and while every contribution is essential, our success will depend on getting a few company supporters capable of contributing larger amounts. But how do you get your organization to become a supporter? Here are some steps you can take to start the conversation and turn your company into a co.up supporter.
+  For co.up to survive, it's going to take recurring support, and while every contribution is essential, our success will depend on getting a few company supporters capable of contributing larger amounts. But how do you get your organization/employer to become a supporter? Here are some steps you can take to start the conversation and turn your company into a co.up supporter.
 </p>
 
 <h3>You were hired there, tell them your co.up story!</h3>
 
 <p>
-  Nothing beats a personal connection. Open your conversation by speaking about how important co.up was in your history and where you think you’d be without it. Was your first meetup here? Did you make any important connections? Would that have been possible without a free community space like co.up?
+  Nothing beats a personal connection. Open your conversation by speaking about how important co.up was in your history and where you think you'd be without it. Was your first meetup here? Did you make any important connections? Would that have been possible without a free community space like co.up?
```

Comment:
@carmem-s All the apostrophes switched to single quotes, can you change it back?